### PR TITLE
Adds 2016 structure to the conf.py file

### PR DESCRIPTION
**Pull request by lamagnifica**
Also random things I typed while looking.
@pdesjardins @srpearce @catong

### CODE REVIEW

**Comment by catong** [comment]
Looks good! :+1:

File: en_us/release_notes/source/2016/website/website_0106_2016.rst [hunk 1]
```diff
@@ -1,3 +1,4 @@
 
-* The latest course in the edX Learning Series, :ref:`Creating Video for the edX
```
Comment (srpearce) [inline en_us/release_notes/source/2016/website/website_0106_2016.rst:2]: I fixed this. I very deliberately fixed this. How did it re-enter the file? Sigh.

File: en_us/release_notes/source/2016/index.rst [hunk 2]
```diff
@@ -4,7 +4,7 @@
 edX Release Notes 2016
 ####################################
 
-The following pages describe what is new in 2016.
+The following pages summarize what is new in 2016.
```
Comment (srpearce) [inline en_us/release_notes/source/2016/index.rst:7]: I know I made this change in the original draft. Aargh.

File: en_us/release_notes/source/conf.py [hunk 3]
```diff
@@ -15,10 +15,16 @@
 project = u'EdX Release Notes'
 
 #remove directory when content is first added to it, and add to index
-exclude_patterns = ['links.rst', 'reusables/*', '2015/analytics/*2015.rst', 
+exclude_patterns = ['links.rst', 'reusables/*', '2016/analytics/*2016.rst',
```
Comment (srpearce) [inline en_us/release_notes/source/conf.py:18]: Thanks for fixing this. I never would have found it.

**Pull request merged**